bin/dfx-snapshot-stock-make

```diff
@@ -20,6 +20,12 @@ source "$(clap.build)"
 : Set up SNS state and create one finalized SNS
 dfx-sns-demo
 
+: Set up ckbtc canisters
+dfx-ckbtc-import --prefix ckbtc_
+dfx-ckbtc-import --prefix ckbtc_test_
+dfx-ckbtc-deploy --prefix ckbtc_
+dfx-ckbtc-deploy --prefix ckbtc_test_
+
```

Comment:
@lmuntaner - The [ckbtc deployment script in the nns-dapp repo](https://github.com/dfinity/nns-dapp/blob/main/scripts/ckbtc/deploy-ckbtc) deploys with the prefix `ckbtc_test_` but dfx.json has the prefix `ckbtc_` (no test).  Which would you like, or both?

Reply:
no `test`. I thought I removed that to be honest.

Thanks for doing this.

DOn't you need to add them to dfx.json?

Reply:
`dfx-ckbtc-import` adds entries to dfx.json, if they are not already there.  If there are already entries, they are used.